lib/src/store_watcher.dart

```diff
@@ -121,9 +126,8 @@ abstract class StoreWatcherMixin implements State<dynamic> { // ignore: TYPE_ARG
     // TODO(abarth): We cancel our subscriptions in [dispose], which means we
     // shouldn't receive this callback when we're not mounted. If that's the
     // case, we should change this check into an assert that we are mounted.
-    if (!mounted)
-      return;
-    setState(() { });
+    if (!mounted) return;
```

Comment:
this 2 changes are only caused by dartfmt. How can I try to improve it , any advice ?